content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -60,7 +60,7 @@ state for some duration:
 
 ## Signers
 
-All signers should provide information about how they work so that clients can predict what will happen to their CSRs.
+Custom signerNames can also be specified. All signers should provide information about how they work so that clients can predict what will happen to their CSRs.
```

Comment:
Hi @mengjiao-liu .
Does this mean that you can a custom `signerName` value or add a custom field?
Is it possible to set multiple values?

Reply:
Please see this comment https://github.com/kubernetes/kubernetes/issues/98146#issuecomment-763669437